addons/vehicles/CfgVehicles/Viper.inc

```diff
@@ -0,0 +1,191 @@
+    receiveRemoteTargets=1;
+    class Components: Components {
+        class SensorsManagerComponent {
+            class Components {
+                class DataLinkSensorComponent: SensorTemplateDataLink { };
+                class PassiveRadarSensorComponent: SensorTemplatePassiveRadar {
+                    class AirTarget {
+                        minRange=8000;
+                        maxRange=8000;
+                        objectDistanceLimitCoef=-1;
+                        viewDistanceLimitCoef=-1;
+                    };
+                    class GroundTarget {
+                        minRange=8000;
+                        maxRange=8000;
+                        objectDistanceLimitCoef=-1;
+                        viewDistanceLimitCoef=-1;
+                    };
+                };
+            };
+        };
+    };
+
+    class Turrets: Turrets {
+        class MainTurret: MainTurret {
+            class Turrets: Turrets {
+                class CommanderOptics: CommanderOptics {
+                    weapons[] = {
+                        "HMG_127_APC",
+                        "Laserdesignator_mounted",
+                        "SmokeLauncher"
+                    };
+                    magazines[] = {
+                        "200Rnd_127x99_mag_Tracer_Yellow",
+                        "200Rnd_127x99_mag_Tracer_Yellow",
+                        "200Rnd_127x99_mag_Tracer_Yellow",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "Laserbatteries"
+                    };
+                    class Components {
+                        class VehicleSystemsDisplayManagerComponentLeft: VehicleSystemsTemplateLeftCommander {
+                            class Components: components {
+                                class SensorDisplay {
+                                    componentType="SensorsDisplayComponent";
+                                    range[]={4000,2000,1000,8000};
+                                    resource="RscCustomInfoSensors";
+                                };
+                            };
+                        };
+                        class VehicleSystemsDisplayManagerComponentRight: VehicleSystemsTemplateRightCommander {
+                            defaultDisplay="SensorDisplay";
+                            class Components: components {
+                                class SensorDisplay {
+                                    componentType="SensorsDisplayComponent";
+                                    range[]={4000,2000,1000,8000};
+                                    resource="RscCustomInfoSensors";
+                                };
+                            };
+                        };
+                    };
+                };
+            };
+            weapons[] = {
```

Comment:
Please move static definitions above class extensions, it makes it hard to track what class they belong to if they're below due to the shifting indentation. e.g.
```
class Foo
{
  bar = 1;
  class Baz
  {
    qux = 1;
  };
};
```